packages/client/src/assets/team/index.ts

```diff
@@ -0,0 +1,6 @@
+import Anastasia from './Anastasia.png'
+import Arthur from './Arthur.png'
+import Iskander from './Iskander.png'
+import Leila from './Leila.png'
+
+export { Anastasia, Arthur, Iskander, Leila }
```

Comment:
Лучше в имени дать понять, что это изображение. А то мало ли, что мы импортируем с такими именами)

```suggestion
export { AnastasiaImage, ArthurImage, IskanderImage, LeilaImage }
```